docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md

```diff
@@ -0,0 +1,54 @@
+# Installing Intel Data Center Attestation Primitives (DCAP) and its dependencies on Windows
+
+
+## [Intel Platform Software for Windows (PSW) v2.4](http://registrationcenter-download.intel.com/akdlm/irc_nas/15654/Intel%20SGX%20PSW%20for%20Windows%20v2.4.100.51291.exe)
+
+After unpacking the self-extracting ZIP executable, install the *PSW_EXE_RS2_and_before* version:
+```cmd
+C:\Intel SGX PSW for Windows v2.3.100.49777\PSW_EXE_RS2_and_before\Intel(R)_SGX_Windows_x64_PSW_2.3.100.49777.exe
+```
+
+## [Azure DCAP client for Windows](https://github.com/Microsoft/Azure-DCAP-Client/tree/master/src/Windows) [optional]
+
+Note that this is optional since you can choose an alternate implementation of THE DCAP client or create your own.
```

Comment:
Nit: "THE" - > "the". I read it as an acronym based on capitalization, but not sure if intentional.

Reply:
Fixed.